cloud/sshclient/sshclient.go

```diff
@@ -38,7 +40,9 @@ func (c *Client) Exec(remoteCmd string) ([]byte, error) {
 }
 
 func (c *Client) cmd() *exec.Cmd {
-	cmd := exec.Command("ssh", destination(c.Username, c.Hostname))
+
+	remoteCmd := fmt.Sprintf("bash -l -c \"start_devbox_shell.sh %s\"", c.ProjectDirName)
```

Comment:
no no no, this is wrong :(

`cmd()` is called from multiple spots. I need to pass the `bash -l ...` remote command into the `exec()` function above.